pyocd/probe/pydapaccess/interface/pyusb_backend.py

```diff
@@ -282,6 +282,10 @@ def __call__(self, dev):
         except (IndexError, NotImplementedError) as error:
             LOG.debug("Error accessing USB device (VID=%04x PID=%04x): %s", dev.idVendor, dev.idProduct, error)
             return False
+        except ValueError as error:
+            # Ignore devices causing ValueError, these are most likely caused by langid issues.
+            LOG.warning("ValueError accessing USB device: %s", error)
+            return False
 
         if device_string is None:
```

Comment:
Here, it seems to want to handle the situation, when product string is not available, but expects that there is no exception (retriever function silently returns `None`).

Reply:
Yep.